contracts/drafts/meta-tx/GSNRecipient.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.5.0;
+
+import "./IRelayRecipient.sol";
+import "./GSNContext.sol";
+
+/*
+ * @dev Base GSN recipient contract, adding the recipient interface and enabling
+ * GSN support. Derived contracts must implement all interface methods.
+ */
+contract GSNRecipient is IRelayRecipient, GSNContext {
+    uint256 constant private RelayedCallAccepted = 0;
+    uint256 constant private RelayedCallRejected = 11;
+
+    function _acceptRelayedCall() internal pure returns (uint256) {
```

Comment:
@frangio any thoughts on these? I tried multiple approaches to see if we could arrive at a nice API, but dislike everything I've arrived at.